test/BUILD.gn

```diff
@@ -568,6 +568,7 @@ test("brave_browser_tests") {
     "//brave/chromium_src/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view_browsertest.cc",
     "//brave/chromium_src/components/content_settings/core/browser/brave_content_settings_registry_browsertest.cc",
     "//brave/chromium_src/third_party/blink/public/platform/disable_client_hints_browsertest.cc",
+    "//brave/chromium_src/third_party/blink/renderer/core/frame/navigator_user_agent_browsertest.cc",
```

Comment:
Curious why put the new test into chromium overrides? Does it make sense to actually override?

Reply:
Override is so that actual architecture, model, platform, etc are not used (and are blank)

Reply:
Typing that out, I think I should put a test to make sure the other values are empty string

Reply:
> Maybe I should put a test to make sure the other values are empty string? WDYT?
IMO makes sense.

I'm asking whether we should make this new file just a normal separate test, not an override. Imagine that one day we will enable chromium tests - and the overriden ones would stay hidden. But they could have some useful bits.

Reply:
Looks like we have several tests already inside `chromium_src`. I see your concern though- if the file is named the same, it won't use the chromium version of the test if we wanted to run `browser_tests` instead of `brave_browser_tests`